### PR TITLE
fixes #38; error in emacs-nox

### LINKED ISSUE #38: undefined function causes emacs config to fail in emacs-nox
`scroll-bar-mode` is undefined in emacs-nox. this causes the emacs config to fail.

### PR DESCRIPTION
**Pull request by samontea**
@rye check this. i doubt there will be any problems

### CODE REVIEW

File: .emacs.d/lisp/visual/frame-c.el [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ;; disable tool bar and menu bar
 (tool-bar-mode -1)
 (menu-bar-mode -1)
-(scroll-bar-mode -1)
+(if (boundp 'scroll-bar-mode)
```
Comment (rye) [inline .emacs.d/lisp/visual/frame-c.el]: Should this be `fboundp`?

Comment (samontea) [reply .emacs.d/lisp/visual/frame-c.el]: yes according to our style guides, but it looks like there's not a difference. http://www.gnu.org/software/emacs/manual/html_node/elisp/Function-Cells.html

Comment (rye) [reply .emacs.d/lisp/visual/frame-c.el]: Would you mind adding the f just so that we're maintaining the stuff?

Comment (samontea) [reply .emacs.d/lisp/visual/frame-c.el]: yeah. googling `fboundp` was all I could do at the gym. 😘

**Review by rye — APPROVED** [review]
Excellent. LGTM.

**Comment by cg505** [comment]
babe funquote that shit

**Comment by rye** [comment]
Oh shoot, I missed that.

**Comment by cg505** [comment]
nbd

**Pull request merged**